src/GLM.jl

```diff
@@ -12,14 +12,15 @@ module GLM
     import Statistics: cor
     import StatsBase: coef, coeftable, confint, deviance, nulldeviance, dof, dof_residual,
                       loglikelihood, nullloglikelihood, nobs, stderror, vcov, residuals, predict,
-                      fitted, fit, model_response, response, modelmatrix, r2, r², adjr2, adjr², PValue
+                      fitted, fit, model_response, response, modelmatrix, r2, r², adjr2, adjr², PValue,
+                      aic, aicc, bic
```

Comment:
No need to import these as we don't reexport them nor overload them for now. We'll have to revisit this later anyway.
```suggestion
                      fitted, fit, model_response, response, modelmatrix, r2, r², adjr2, adjr², PValue
```

Reply:
Ah sorry I wanted to commit this change directly but I don't have rights to edit your PR.